src/jit/gentree.cpp

```diff
@@ -7489,14 +7489,15 @@ GenTree* Compiler::gtCloneExpr(
         case GT_ARR_ELEM:
         {
             GenTree* inds[GT_ARR_MAX_RANK];
-            for (unsigned dim = 0; dim < tree->gtArrElem.gtArrRank; dim++)
+            for (unsigned dim = 0; dim < tree->AsArrElem()->gtArrRank; dim++)
```

Comment:
Could you please create a temp for it:
`GenTreeArrElem* arrElem = tree->AsArrElem();` 
and use it instead of `tree->AsArrElem()` below?